src/app/register/pages/register/register.component.ts

```diff
@@ -137,5 +138,28 @@ export class RegisterComponent implements OnInit {
     if (event.previouslySelectedIndex === 0) {
       this.afterStepASubmitted()
     }
+    if (this.platform.columns4 || this.platform.columns8) {
+      this.focusCurrentStep(event)
+    }
+  }
+
+  // Fix to material vertical stepper not focusing current header
+  // related issue https://github.com/angular/components/issues/8881
+  focusCurrentStep(event: StepperSelectionEvent) {
+    let nextStep: ElementRef
+    if (event.selectedIndex === 0) {
+      nextStep = this.stepComponentA
+    } else if (event.selectedIndex === 1) {
+      nextStep = this.stepComponentB
+    } else if (event.selectedIndex === 2) {
+      nextStep = this.stepComponentC
+    }
+    // On mobile scroll the current step component into view
+    if (this.platform.columns4 || this.platform.columns8) {
+      setTimeout(() => {
+        // tslint:disable-next-line: whitespace
+        ;(nextStep.nativeElement as HTMLElement).scrollIntoView()
```

Comment:
I think it is better if you assign `nextStep.nativeElement as HTMLElement` to a constant and after you call `scrollIntoView()` instead of disabling tslint and it's more readable